Documentation/policy/intro.rst

```diff
@@ -57,6 +57,8 @@ Similarly, you can enable the policy enforcement mode across a Kubernetes cluste
     - name: CILIUM_ENABLE_POLICY
       value: always
 
+If you want to configure health entity to check cluster-wide connectivity when you start cilium-agent with ``enable-policy=always``,
+see :ref:`health_endpoint`
```

Comment:
Nitpick: Period at the end of the sentence. Could you also please wrap the line on ~80 characters? Although this document is not really consistent on this, most of the documentation does it.

```suggestion
If you want to configure health entity to check cluster-wide connectivity when
you start cilium-agent with ``enable-policy=always``, see :ref:`health_endpoint`.
```

Reply:
I think we should move this note right after the `always` paragraph on line 33. Otherwise, users may miss it.

Also, the cluster will not be reported as healthy without this policy, so maybe we could reword this as something like: `With that mode you will likely want to enable communications to and from the health endpoint. See ...`. WDYT?

Reply:
I agree with this idea. Fixed them.